scripts/test_runner.sh

```diff
@@ -62,8 +62,7 @@ if [ -n "${TEST_DIR}" ]; then
              [ -n "${APPLY_WORKLOAD}" ] && OPTS+=(-applyWorkload "${APPLY_WORKLOAD}")
              [ -n "${DELETE_WORKLOAD}" ] && OPTS+=(-deleteWorkload "${DELETE_WORKLOAD}")
              [ -n "${DESCRIPTION}" ] && OPTS+=(-description "${DESCRIPTION}")
-             [ -n "${DEBUG_MODE}" ] && OPTS+=(-debug "${DEBUG_MODE}")
-          go test "${OPTS[@]}"
+      go test "${OPTS[@]}"
```

Comment:
It looks like we lost the DEBUG_MODE line. Can we add that back in? `[ -n "${DEBUG_MODE}" ] && OPTS+=(-debug "${DEBUG_MODE}")`